test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -51,6 +52,42 @@ public void PushCommand_PushToV2FileSystemSource()
             }
         }
 
+        // Tests pushing multiple packages (multiple paths)
+        [Fact]
+        public void PushCommand_PushMultiplePaths()
+        {
+            var nugetexe = Util.GetNuGetExePath();
+
+            using (var packageDirectory = TestDirectory.Create())
+            using (var source = TestDirectory.Create())
+            {
+                // Arrange
+                var packagePaths = new[]
+                {
+                    Util.CreateTestPackage("testPackageA", "1.1.0", packageDirectory),
+                    Util.CreateTestPackage("testPackageB", "1.1.0", packageDirectory),
+                };
+
+                // Act
+                var args = new[] {"push", "-Source", source}.Concat(packagePaths);
+                var result = CommandRunner.Run(
```

Comment:
Explicit parameter names would be awesome. I know most of these tests don't do that (some have a comment), but it'd fit with our new coding guidelines.